pulsar-broker/src/test/java/org/apache/pulsar/broker/auth/MockAuthenticationProvider.java

```diff
@@ -65,4 +71,41 @@ public String authenticate(AuthenticationDataSource authData) throws Authenticat
                 "Not a valid principle. Should be [pass|fail|error].[pass|fail|error], found " + principal);
     }
 
+    @Override
+    public AuthenticationState newAuthState(AuthData authData, SocketAddress remoteAddress, SSLSession sslSession)
```

Comment:
With the addition of `MockMutableAuthenticationProvider`, I think we can remove this method and the `MockAuthState` class, is that correct?

Reply:
Sure, I forget to remove this method.